apps/backoffice-v2/src/lib/blocks/hooks/useDocumentBlocks/useDocumentBlocks.tsx

```diff
@@ -388,9 +388,32 @@ export const useDocumentBlocks = ({
                         value,
                         formatMinimum,
                         formatMaximum,
+                        default: defaultValue,
                       },
                     ]) => {
-                      const fieldValue = value || (properties?.[title] ?? '');
+                      const getFieldValue = () => {
+                        if (typeof value !== 'undefined') {
+                          return value;
+                        }
+
+                        if (
+                          typeof properties?.[title] === 'undefined' &&
+                          typeof defaultValue !== 'undefined'
+                        ) {
+                          return defaultValue;
+                        }
+
+                        if (typeof properties?.[title] === 'undefined' && type === 'boolean') {
+                          return false;
+                        }
+
+                        if (typeof properties?.[title] === 'undefined') {
+                          return '';
+                        }
```

Comment:
Made all checks in a way that won't fail on things like `0`, `false`, empty strings and so on. `null` is actually a valid value especially since some fields could be nullable.